docs/modules/control/inverted_pendulum_lqr_control/inverted_pendulum_mpc_control.rst

```diff
@@ -0,0 +1,6 @@
+Inverted Pendulum LQR Control
+-----------------------------
+
+Bipedal Walking with modifying designated footsteps, using LQR controller
```

Comment:
This sentence is wrong, this is not for footsteps planner. Same issue is in the MPC controller too.